docs/cookbook/paginating_api_search_results.md

```diff
@@ -1,17 +1,15 @@
 # Paginating API search results
 
-## Description
+When listing content (e.g. blog posts), pagination is a very common use case and can be difficult to implement by hand.
```

Comment:
I would like to say: "let the robot/computer do the work for us", "do not reinvent the wheel" in this intro.

Reply:
Rephrased in line below.